### DISCUSSION TITLE
Maintain persistent sessions

### DISCUSSION THREAD

**Discussion by tyagirajat200** [eclipse-theia/theia#11720] [Q&A]
How we can maintain persistent sessions. If we open multiple tabs across devices then all the worspace settings, layout, terminals etc should be persistent as currently theia maintain persistent sessions in same window if we refresh the tab any time.

**Comment by vince-fugnitto** [comment]
@tyagirajat200 thank you for the discussion, Theia does not handle this or workspace management natively so it would need to be implemented on your end based on your needs and use-case.
